redis/src/r2d2.rs

```diff
@@ -3,6 +3,11 @@ use crate::sentinel::LockedSentinelClient;
 use crate::types::closed_connection_error;
 use crate::{ConnectionLike, RedisError};
 
+// NOTE: GenericConnection and generic friends
```

Comment:
generic friends? what does that mean?

Reply:
Hmm probably a bad habit from mine to word it like that. Not even sure what I mean anymore. I think I meant to say that if we do the same thing for async that it would also not be supported here, but than again wouldn't be relevant anyway as I don't think this is ever used for that purpose?

Reply:
no, the r2d2 implementation AFAIK doesn't work with async connections, and it seems there's no public request for that :)

Reply:
Removed comment as this is no longer an issue, it can be requested if ever needed.